src/main/java/io/vertx/openapi/validation/Parameter.java

```diff
@@ -18,13 +18,23 @@
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
+import java.util.function.Supplier;
+
 interface Parameter {
   /**
    * @return null if value is not a {@link String}, otherwise it returns value
    */
   @Nullable
   default String getString() {
-    return isString() ? (String) get() : null;
+    return getString(() -> null);
+  }
+
+  /**
+   * @return the default supplied by a {@link java.util.function.Supplier} if value is not a {@link String} or is
+   * null, otherwise it returns the value.
+   */
+  default String getString(Supplier<String> defaultValue) {
```

Comment:
After the discussion with Julien, we agreed to also remove the Supplier. 
```suggestion
  default String getString(String defaultValue) {
```

Reply:
Removed the usages of Supplier now.